src/utils/types.ts

```diff
@@ -44,3 +49,13 @@ export type MetadataRetrieveResultJson = Omit<MetadataApiRetrieveStatus, 'zipFil
 };
 
 export type RetrieveResultJson = (MetadataApiRetrieveStatus & { files: FileResponse[] }) | MetadataRetrieveResultJson;
+
```

Comment:
Is there more appropriate place for these functions?

Reply:
I like keeping type guards with types...what have you got in mind?

Reply:
I am good, just curious